.gitignore

```diff
@@ -3,5 +3,4 @@ build
 bin
 .classpath
 .project
-.settings
-.vscode
```

Comment:
Removed `.vscode` from `.gitignore` because the repository coding standards restrict the use of star imports which an editor may produce.